src/module-elasticsuite-core/Client/ClientBuilder.php

```diff
@@ -109,6 +109,17 @@ public function build($options = [])
             $clientBuilder->setSelector($selector);
         }
 
+        $connectionParams = ['client' => []];
+        if (isset($options['timeout'])) {
+            $connectionParams['client']['timeout'] = $options['timeout'];
```

Comment:
Change $options['timeout'] to $options['read_timeout'] here also to be consistent ?